css/styles.css

```diff
@@ -1,715 +1,1542 @@
-		display: none;
-	}
-	
-	.sneakers_margin {
-		margin: 0 0 0 50px;
-	}
-	
-	.sneakers__photo {
-		width: 164px;
-    		height: 153px;
-    		padding: 0 0 0 49px;
-	}
-	
-	.sneakers__name {
-		font-size: 15px;
-		padding: 7px 0 0 3px;
-		margin-bottom: 3px;
-	}
-	
-	.prise__text {
-		font-size: 20px;
-		padding: 0 0 0 4px;
-	}
-	
-	.prise-circles {
-		margin: 0 -57px 0 0;
-	}
-	
-	.circle {
-		width: 15px;
-		height: 15px;
-		margin: 0 0 0 5px;
-	}
+  .section-stories {
+    padding: 58px 0 0 0;
+    height: 775px;
+    margin-bottom: 281px;
+  }
+
+  .section-stories::after {
+    bottom: -18px;
+    height: 18px;
+  }
+
+  .section-stories__title {
+    font-size: 20px;
+    margin: 0 0 27px 0;
+  }
+
+  .stories-content {
+    margin: 0 23px 0 0;
+  }
+
+  .stories-content__photo {
+    width: 279px;
+    height: 148px;
+  }
+
+  .stories-content__title {
+    font-size: 20px;
+    margin: 24px 0 7px 0;
+  }
+
+  .stories-content__text {
+    font-size: 14px;
+    line-height: 18px;
+    width: 276px;
+    margin: 0 0 24px 0;
+  }
+
+  .more__link {
+    font-size: 12px;
+    width: 136px;
+  }
+
+  .more__link::after {
+    width: 40px;
+    height: 9px;
+    top: 1px;
+  }
+
+  .more__data {
+    margin: -3px 21px 0 0;
+  }
+
+  .stories-bottom__hashtag {
+    font-size: 25px;
+    padding: 12px 53px 0 0;
+  }
+
+  .pictures__image {
+    width: 140px;
+    height: 140px;
+  }
+
+  .pictures__image:nth-child(3) {
+    order: 2;
+  }
+
+  .pictures__image:nth-child(5) {
+    order: 1;
+  }
 }
 
 @media screen and (max-width: 320px) {
-	.section-products {
-		width: 320px;
-	}
-	
-	.ad {
-		width: 100%;
-		height: 191px;
-		background: url("../images/section1/ad/main374.svg") center no-repeat;
-		background-size: cover;
-	}
-	
-	.ad__title {
-		padding: 66px 0 0 20px;
-	}
-	
-	.ad__link {
-		padding: 0 54px 0 20px;
-	}
-	
-	.sneakers_margin {
-		margin: 0 0 0 23px;
-	}
-	
-	.prise__text {
-		padding: 0 0 0 3px;
-	}
-	
-	.prise-circles {
-		margin: 0 -56px 0 0;
-	}
+  .section-stories-top {
+    padding: 0 0 0 25px;
+  }
+
+  .more__data {
+    margin: -3px 1px 0 0;
+  }
+
+  .stories-bottom__hashtag {
+    padding: 12px 26px 0 0;
+  }
+}
+
+/* Footer */
+.footer {
+  max-width: 1170px;
+  color: #fff;
+  background-color: #000;
+  font-size: 12px;
+  margin: 0 auto;
+}
+
+.footer-container {
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+}
+
+.footer-top {
+  display: flex;
+  padding: 66px 0 37px 4px;
+}
+
+.footer-box {
+  display: flex;
+  flex-direction: column;
+}
+
+.footer-box:first-child {
+  margin: 0 114px 0 0;
+}
+
+.footer-box:nth-child(2) {
+  margin: 0 72px 0 0;
+}
+
+.footer-box:nth-child(3) {
+  margin: 0 85px 0 0;
+}
+
+.footer-box__text {
+  text-transform: uppercase;
+  font-weight: bold;
+  margin: 0 0 8px 0;
+}
+
+.footer-box:last-child .footer-box__text {
+  margin-bottom: 5px;
+}
+
+.footer-box__link {
+  color: #fff;
+  text-decoration: none;
+  margin: 0 0 5px 0;
+}
+
+.footer-box:last-child 
```

Comment:
Codacy found an issue: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/Itogovya_axion/pullRequest?prid=6118030)